src/templates/settings.html

```
@@ -205,138 +279,250 @@ <h3>Software Update</h3>
+                                    <div class="logs-header">
+                                        <div class="title-container">
+                                            <!-- Inline SVG log icon (document-like) to differentiate from gear -->
+                                            <svg class="app-icon logs-icon" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+                                                <path class="icon-fill" d="M7 3h7l5 5v13a1 1 0 0 1-1 1H7a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2z" stroke="currentColor" stroke-width="1.6"/>
+                                                <path d="M14 3v4a1 1 0 0 0 1 1h4" stroke="currentColor" stroke-width="1.6"/>
+                                                <path class="icon-accent" d="M8 12h8M8 16h8M8 20h8" stroke="currentColor" stroke-width="1.6"/>
+                                            </svg>
+                                            <h4 class="app-title">Logs</h4>
+                                        </div>
+                                        <div class="logs-filters">
+                                            <input id="logsFilter" type="text" placeholder="Filter" class="form-input" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)